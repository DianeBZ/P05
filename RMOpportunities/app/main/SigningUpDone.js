import React, { Component } from 'react';
import {
	StyleSheet,
	Text,
	View,
	TouchableHighlight,
    Dimensions,
} from 'react-native';

var width = Dimensions.get('window').width;
var height = Dimensions.get('window').height;

import Translation from './Translation';

export default class SigningUpDone extends Component {
  render() 
  {
    return (
      <View style={styles.container1}>
		<View style={{height:height*0.08}}/>
        <Text style={styles.titre}>
			{Translation[lang].inscription_r√©ussie}
		</Text>
		<View style={styles.subContainer}>
			<Text style={styles.bienvenue}>
				{Translation[lang].bienvenue_RMO}
			</Text>
			<View style={styles.sContainerPre}>
				<Text style={styles.presentation}>
					{Translation[lang].achetez_vendez}
				</Text>
			</View>
			<View style={styles.sContainerBouton}>
            <TouchableHighlight style={[styles.button, {width: width *0.25, backgroundColor:'#A4D04A'}]} onPress={this._onPressBouton} underlayColor="rgb(138,183,46)">
              <Text style={styles.buttonText}>
				  {Translation[lang].accueil}
              </Text>
            </TouchableHighlight>
          </View>
		</View>
      </View>
    );
  }
  //Navigation for this page
  _onPressBouton = () => {
      const { navigator } = this.props;
        if (navigator) {
            navigator.popToTop(); // Pop to the first scene in the stack, unmounting every other scene
        }
  }
}

const styles = StyleSheet.create({
	
  container1: {
    flex: 1,
    justifyContent: 'flex-start',
    alignItems: 'center',
    backgroundColor: '#F2F2F2',
  },
  titre:{
       fontSize: 32,
       textAlign: 'center',
       color: '#000000',
       marginTop: 15,
       marginBottom: 15
   },
  subContainer:{
    backgroundColor:'#FFFFFF',
    flex: 1,
  },
  sContainerPre:{
    backgroundColor:'#A4D04A',
    height: height*0.16,
    justifyContent:'center',
  },
  sContainerBouton:{
    flex:1,
    justifyContent: 'flex-start',
    alignItems: 'flex-end',
  },
  bienvenue: {
	color : '#A4D04A',
	fontWeight:'bold',
	textAlign:'center',
	fontSize: 25,
	margin:10,
  },
  presentation: {
    textAlign: 'center',
    color:"black",
    fontSize: 20,
  },
  button:{
        height: height*0.05,
        marginTop: 20,
        borderRadius:5,
        justifyContent: 'center',
    },
  buttonText:{
        color: "white",
        textAlign: "center",
        fontSize: 18,
    }
});