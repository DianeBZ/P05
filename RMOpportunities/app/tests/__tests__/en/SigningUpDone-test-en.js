// __tests__/SigningUpDone-test-en.js
import 'react-native';
import React from 'react';

global.lang = 'en';
import SigningUpDone from '../../../main/SigningUpDone';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <SigningUpDone />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
