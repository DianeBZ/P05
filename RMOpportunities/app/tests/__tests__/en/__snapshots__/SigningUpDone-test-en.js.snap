// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F2F2F2",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <View
    style={
      Object {
        "height": 106.72,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#000000",
        "fontSize": 32,
        "marginBottom": 15,
        "marginTop": 15,
        "textAlign": "center",
      }
    }
  >
    Successful registration !
  </Text>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#A4D04A",
          "fontSize": 25,
          "fontWeight": "bold",
          "margin": 10,
          "textAlign": "center",
        }
      }
    >
      Welcome to RMOpportunities
    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#A4D04A",
          "height": 213.44,
          "justifyContent": "center",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "black",
            "fontSize": 20,
            "textAlign": "center",
          }
        }
      >
        Buy and sell your surplus stocks or raw chemical materials
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "flex-end",
          "flex": 1,
          "justifyContent": "flex-start",
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            Array [
              Object {
                "borderRadius": 5,
                "height": 66.7,
                "justifyContent": "center",
                "marginTop": 20,
              },
              Object {
                "backgroundColor": "#A4D04A",
                "width": 187.5,
              },
            ],
          ]
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "fontSize": 18,
              "textAlign": "center",
            }
          }
        >
          Homepage
        </Text>
      </View>
    </View>
  </View>
</View>
`;
